packages/vue/src/components/organisms/SfProductCard/SfProductCard.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <div class="sf-product-card">
     <div class="sf-product-card__image-wrapper">
-      <SfLink class="sf-product-card__link" :link="link">
+      <component
```

Comment:
@martaradziszewska maybe we can just move SfLink with an image to slot? It should be easier to replaced? WDYT?

Reply:
that is also a good idea, I'll try this :)